### PR TITLE
Feature/expansion

### PR DESCRIPTION
**Pull request by jsp-up**

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -3,13 +3,15 @@ import { NgModule } from '@angular/core';
 
 import { AppComponent } from 'src/app/app.component';
 import { AppRoutingModule } from 'src/app/app-routing.module';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
```
Comment (MichaelGrosio) [inline src/app/app.module.ts]: Imports

File: projects/kalidea/kaligraphi/src/lib/molecules/kal-accordion/kal-accordion.component.ts [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  }
+
+  /**
+   * Handles keyboard events coming in from the panel headers.
+   */
+  handleHeaderKeydown(event: KeyboardEvent) {
+    const {keyCode} = event;
+    const manager = this.keyManager;
+
+    if (keyCode === HOME) {
```
Comment (MichaelGrosio) [inline projects/kalidea/kaligraphi/src/lib/molecules/kal-accordion/kal-accordion.component.ts]: switch

File: projects/kalidea/kaligraphi/src/lib/molecules/kal-accordion/kal-expansion-panel-header/kal-expansion-panel-header.component.ts [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+  /**
+   * Gets the expanded state string of the panel.
+   */
+  getExpandedState(): string {
+    return this.panel.getExpandedState();
+  }
+
+  /** Gets whether the expand indicator should be shown. */
+  showToggle(): boolean {
+    return !this.panel.hideToggle && !this.panel.disabled;
```
Comment (MichaelGrosio) [inline projects/kalidea/kaligraphi/src/lib/molecules/kal-accordion/kal-expansion-panel-header/kal-expansion-panel-header.component.ts]: isPanelDisabled()

File: projects/kalidea/kaligraphi/src/lib/molecules/kal-accordion/kal-expansion-panel-header/kal-expansion-panel-header.component.ts [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+    switch (event.keyCode) {
+      // Toggle for space and enter keys.
+      case SPACE:
+      case ENTER:
+        event.preventDefault();
+        this.toggle();
+        break;
+      default:
+        if (this.panel.accordion) {
+          console.log(this.panel);
```
Comment (MichaelGrosio) [inline projects/kalidea/kaligraphi/src/lib/molecules/kal-accordion/kal-expansion-panel-header/kal-expansion-panel-header.component.ts]: console.log

**Review by MichaelGrosio — CHANGES_REQUESTED** [review]

File: projects/kalidea/kaligraphi/package.json [hunk 5]
```diff
@@ -13,6 +13,8 @@
     "@angular/common": "^6.0.0-rc.0 || ^6.0.0",
     "@angular/core": "^6.0.0-rc.0 || ^6.0.0",
     "@angular/cdk": "^6.4.7",
+    "@angular/animations": "^6.1.0",
+    "@angular/cdk": "^6.4.7",
```
Comment (ydomenjoud) [inline projects/kalidea/kaligraphi/package.json]: 2 times cdk

**Review by MichaelGrosio — APPROVED** [review]

**Pull request merged**